src/iuse.cpp

```diff
@@ -2012,41 +2012,79 @@ int iuse::sew_advanced(player *p, item *it, bool, const tripoint& )
 
     // TODO: Wrap all the mods into structs, maybe even json-able
     // All possible mods here
-    std::array<std::string, 4> clothing_mods{
+    const std::array<const std::string, 4> clothing_mods{
         { "wooled", "furred", "leather_padded", "kevlar_padded" }
     };
 
     // Materials those mods use
-    std::array<std::string, 4> mod_materials{
+	const std::array<const std::string, 4> mod_materials{
```

Comment:
Tabs here: you should replace all tabs with 4 spaces each.
If you're adding `const` everywhere, also add `static` so that they aren't regenerated every time the function is executed.